acceptance/repository-postgresql/src/__tests__/postgresql.datasource.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/test-repository-postgresql
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {CrudFeatures, DataSourceOptions} from '@loopback/repository-tests';
+
+const connector = require('loopback-connector-postgresql');
+
+export const POSTGRESQL_CONFIG: DataSourceOptions = {
+  connector,
+  host: process.env.POSTGRESQL_HOST || 'localhost',
+  port: process.env.POSTGRESQL_PORT || 5432,
+  database: process.env.POSTGRESQL_DATABASE || 'repository-tests',
+  username: process.env.POSTGRESQL_USER || 'root',
+  password: process.env.POSTGRESQL_PASSWORD || 'pass',
+  createDatabase: true,
+};
+
+export const POSTGRESQL_FEATURES: Partial<CrudFeatures> = {
+  idType: 'number',
+  freeFormProperties: false,
+  emptyValue: null,
+  supportsTransactions: false,
```

Comment:
Are you sure that our PostgreSQL connector does not support transactions? The database certainly does support them.

Reply:
I was using `DefaultCrudRepository` before by accident, but I've changed this to `true` now.